src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -50,23 +51,18 @@ export default class AnnotationColumnFormatter
     }
 
    /**
-     * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns 
-     * an empty object.
-     */
+    * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns 
+    * an empty object.
+    */
     public static getCivicEntry(copyNumberData:DiscreteCopyNumberData[], civicGenes:ICivicGene, civicVariants:ICivicVariant): ICivicEntry | null
     {
         let geneSymbol: string = copyNumberData[0].gene.hugoGeneSymbol;
         let geneVariants: {[name: string]: ICivicVariantData} = civicVariants[geneSymbol];
         let geneEntry: ICivicGeneData = civicGenes[geneSymbol];
         let civicEntry = null;
         //Only return data for genes with variants or it has a description provided by the Civic API
-        if (geneVariants ||
-               (geneEntry && geneEntry.description != "")) {
-            civicEntry = {name: geneEntry.name,
-                             description: geneEntry.description,
-                             url: geneEntry.url,
-                             variants: geneVariants,
-                            }
+        if (geneVariants || (geneEntry && geneEntry.description != "")) {
```

Comment:
Well, in mutations we only display the CIViC entries of the genes that have the specific mutation (in geneVariants). However, in CNA we want to display all the genes with an entry with description (with or without CNA variants), and but also the genes with no description but with CNA variants. The genes without CNA variants but with description will have the icon in grey instead of color.